spinner/build.sh

```diff
@@ -48,15 +48,29 @@ function build_service_liferay {
 
 	cp ../../orca/templates/liferay/resources/usr/local/liferay/scripts/pre-startup/10_wait_for_dependencies.sh build/liferay/resources/usr/local/liferay/scripts/pre-startup
 
-	(
-		echo "FROM $(grep -e '^liferay.workspace.docker.image.liferay=' "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/liferay/gradle.properties" | cut -d'=' -f2)"
+	if [ "$(git --git-dir "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/.git" rev-parse --abbrev-ref HEAD)" == "master" ]
```

Comment:
Using --git-dir directive, unfortunately the flags can't be reordered.

Reply:
@pedropaccola, did you mean that we can't do the following, right?
```
git --abbrev-ref HEAD --git-dir "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/.git" rev-parse)
```

Reply:
Right! Both `-C` and `--git-dir` are flags for the `git` command, while `--abbrev-ref` is a flag for `rev-parse` command.